users/decorator.py

```diff
@@ -0,0 +1,35 @@
+import jwt
+import json
+
+from django.http import (
+    JsonResponse,
+    HttpResponse
+)
+
+from users.models import User
+from dr_martens.settings import (
+    SECRET_KEY,
+    ALGORITHM
+)
+
+def login_check(func):
+    def wrapper(self, request, *args, **kwargs):
+        access_token = request.headers.get("Authorization", None)
+        if access_token is not None:
+            try:
+                decode_token  = jwt.decode(access_token, SECRET_KEY, algorithms = ALGORITHM)
+                users_id      = decode_token['users_id']
+                users         = User.objects.get(id = users_id)
+                request.users = users
+                return func(self, request, *args, **kwargs)
+
+            except jwt.DecodeError:
+                return JsonResponse({'message' : 'INVALID_USER'}, status = 400)
+
+            except jwt.ExpiredSignatureError:
+                return JsonResponse({'message' : 'INVALID_USER'}, status = 400)
```

Comment:
```suggestion
                return JsonResponse({'message' : 'TOKEN_EXPIRED'}, status = 401)
```
만료된 토큰이니까 unauthorized 상황이고 메세지는 유저가 잘못됐다기보다는 토큰이 만료됐다고 안내해주세요.